src/SDL_gpu_textengine.c

```diff
@@ -207,13 +206,13 @@ static AtlasTexture *CreateAtlas(SDL_GPUDevice *device)
     SDL_EndGPURenderPass(rpass);
     SDL_SubmitGPUCommandBuffer(cbuf);
 
-    int num_nodes = ATLAS_TEXTURE_SIZE / 4;
+    int num_nodes = atlas_texture_size / 4;
     atlas->packing_nodes = (stbrp_node *)SDL_calloc(num_nodes, sizeof(*atlas->packing_nodes));
     if (!atlas->packing_nodes) {
         DestroyAtlas(device, atlas);
         return NULL;
     }
-    stbrp_init_target(&atlas->packer, ATLAS_TEXTURE_SIZE, ATLAS_TEXTURE_SIZE, atlas->packing_nodes, num_nodes);
+    stbrp_init_target(&atlas->packer, atlas_texture_size, atlas_texture_size, atlas->packing_nodes, num_nodes);
```

Comment:
Can this be a member of the atlas instead of being passed around as a parameter?